### PR TITLE
feat: add aggregate to modelsApi

### PR DESCRIPTION
**Pull request by bkuzma**

### CODE REVIEW

File: packages/alpha/src/types.ts [hunk 1]
```diff
@@ -14,6 +19,11 @@ export * from '@cognite/sdk';
 // This file is here mostly to allow apis to import { ... } from '../../types';
 // Overriding types should probably be done in their respective API endpoint files, where possible
 
+/**
+ * Response from models aggregate endpoint
+ */
+export type ModelAggregate = AggregateResponse;
```
Comment (polomani) [inline packages/alpha/src/types.ts]:
```suggestion
export type SimulatorModelAggregate = AggregateResponse;
```

File: packages/alpha/src/__tests__/api/simulatorModelsApi.spec.ts [hunk 2]
```diff
@@ -54,6 +54,11 @@ describeIf('simulator models api', () => {
     expect(res[0].externalId).toBe(modelExternalId);
   });
 
+  test('aggregate models', async () => {
+    const aggregate_response = await client.simulators.aggregateModels();
```
Comment (polomani) [inline packages/alpha/src/__tests__/api/simulatorModelsApi.spec.ts]:
```suggestion
    const aggregateResponse = await client.simulators.aggregateModels();
```
snake case?

File: packages/alpha/src/types.ts (hunk 1 above)
Comment (bkuzma) [reply packages/alpha/src/types.ts]: fixed

File: packages/alpha/src/__tests__/api/simulatorModelsApi.spec.ts (hunk 2 above)
Comment (bkuzma) [reply packages/alpha/src/__tests__/api/simulatorModelsApi.spec.ts]: Some of the other tests used snake case 🤷 I changed it now to use camel case here.

Comment (polomani) [reply packages/alpha/src/__tests__/api/simulatorModelsApi.spec.ts]: wow, yeah, it was prob just an overlook

File: packages/alpha/src/api/simulators/simulatorsApi.ts [hunk 3]
```diff
@@ -148,6 +148,10 @@ export class SimulatorsAPI extends BaseResourceAPI<Simulator> {
     return this.runsApi.retrieve(ids);
   };
 
+  public aggregateModels = (filter?: SimulatorModelFilterQuery) => {
+    return this.modelsApi.aggregate(filter);
+  };
```
Comment (polomani) [inline packages/alpha/src/api/simulators/simulatorsApi.ts]:
```suggestion
  public aggregateModels = (query?: SimulatorModelAggregateQuery) => {
    return this.modelsApi.aggregate(filter);
  };
```

type is wrong here

Comment (polomani) [reply packages/alpha/src/api/simulators/simulatorsApi.ts]: where SimulatorModelAggregateQuery is

```
export interface SimulatorModelAggregateQuery {
  /**
   * Filter on models with strict matching.
   */
  filter?: SimulatorModelFilterQuery;
  aggregate: "count";
}
```

Comment (polomani) [reply packages/alpha/src/api/simulators/simulatorsApi.ts]: only count is supported for now

Comment (bkuzma) [reply packages/alpha/src/api/simulators/simulatorsApi.ts]: made the update now

File: packages/alpha/src/api/simulators/modelsApi.ts [hunk 4]
```diff
@@ -25,6 +27,12 @@ export class ModelsAPI extends BaseResourceAPI<SimulatorModel> {
     return this.createEndpoint(items);
   };
 
+  public aggregate = (
+    query?: SimulatorModelAggregateQuery
```
Comment (polomani) [inline packages/alpha/src/api/simulators/modelsApi.ts]:
```suggestion
    query: SimulatorModelAggregateQuery
```
query is required in our spec

File: packages/alpha/src/__tests__/api/simulatorModelsApi.spec.ts [hunk 5]
```diff
@@ -54,6 +54,11 @@ describeIf('simulator models api', () => {
     expect(res[0].externalId).toBe(modelExternalId);
   });
 
+  test('aggregate models', async () => {
+    const aggregateResponse = await client.simulators.aggregateModels();
```
Comment (polomani) [inline packages/alpha/src/__tests__/api/simulatorModelsApi.spec.ts]:
```suggestion
    const aggregateResponse = await client.simulators.aggregateModels({ aggregate: "count" });
```
is required for us

File: packages/alpha/src/api/simulators/modelsApi.ts (hunk 4 above)
Comment (bkuzma) [reply packages/alpha/src/api/simulators/modelsApi.ts]: pushed up the change now

File: packages/alpha/src/__tests__/api/simulatorModelsApi.spec.ts (hunk 5 above)
Comment (bkuzma) [reply packages/alpha/src/__tests__/api/simulatorModelsApi.spec.ts]: fixed

**Review by polomani — APPROVED** [review]
lgtm

**Pull request merged**